app/alerts.py

```diff
@@ -330,15 +350,16 @@ def build_alerts_map():
     The following function mobilises functions defined hereabove or in the utils module to
     instantiate and return a dl.Map object, corresponding to the "Alerts and Infrastructure" view.
     """
-    map_object = dl.Map(center=[46.5, 2],     # Determines the point around which the map is initially centered
-                        zoom=6,               # Determines the initial level of zoom around the center point
+    map_object = dl.Map(center=user_department_lat_lon(),  # Determines the point around which the map is centered
```

Comment:
I am not sure to see the interest in having a specific function that defines the center and zoom of the map when instantiating it: as soon as the app is opened, callbacks are triggered that update these properties. Until the user is logged in, these properties are set to arbitrary values and then, they are modified to zoom the map around the right point. 

But I may be missing something here?

Reply:
Oh yeah you're right, did that before the login PR